Getting-Started-with-Akka-http-signature.md

```diff
@@ -0,0 +1,232 @@
+
+@ keyJenPointed = keyGraph.pointer
+
+keyJenPointed: Jena#Node = 5880.026289.0-02638.0-6288.0-06411.0-23110.021178.029228.0
+```
+The user can then choose to view this graph in a different format - like turtle for example. In order to do that however, more external libraries are required:
+
+```scala
+@ import org.w3.banana._
+@ import org.w3.banana.syntax._
+@ import ops._  
+@ import org.w3.banana.jena.Jena
+```
+One can then represent the key's Pointed Graph in turtle: 
+
+```scala
+@ val toTurtle = turtleWriter.asString(keyGraph.graph,"").get
+
+toTurtle: String = """<5880.026289.0-02638.0-6288.0-06411.0-23110.021178.029228.0>
+        <http://www.w3.org/ns/auth/cert#exponent>
+                65537 ;
+        <http://www.w3.org/ns/auth/cert#modulus>
+                "00937adccd722bc982aed4847872b81e36b890bca13166714bc2befe4d8547b6218ecd2da1eb020198a4ea00e4db6757c7dda738ec8db8b3bf211d3a3a17e196a2035bc4c79d06d8a581487d9f49e86374712b10ef500dfa242a20cab52911e2636c9d99b21fe9768ef2381989a25dc8b0b7a46531249aac27c4b8ab451a19d5fbdfa5f78b0deac9778c7ff87cf6106ae4a6433466beb21df1265bf1fc9ab9cc80d7aff8cc4d0f67ae28647e4048da8df753493b8de6a8e0961416b4b37f7012907d2e756034b8a84c6e495c8f1d81f69843ae51379571d83b38e1c08a08c3748ef75ed7ecb016d0c8426b30c8c5060a08f87f6764b0ec14667cd6f0daa1244157"^^<http://www.w3.org/2001/XMLSchema#hexBinary> .
+"""
+``` 
+This will return a String representation of the Pointed Graph in turtle format. This representation can also be saved on your local file system like so:
+
+```scala
+@ write(wd/"publicKey.ttl", RSAKeys.save(toTurtle))
+```
+As is evident, the header is a link, which can be quite difficult to process. Because of this, the user can change it by using a file editor into something simpler - such as: *<#key>*
+
+### Attaching Public keys to a File/URI
+
+One can use the cp or the mv Ammonite commands to move the public key file into the test_www directory which resides within the rww-play directory. The process of attaching the file, containing the key to a URI is very similar.
+
+```scala
+@ cp(wd/"publicKey.ttl" , wd/"rww-play"/"test_www"/"pubKey.ttl")
+```
+
+The Akka-http-signature library makes use of symbolic links when accessing the public keys. One can make a symbolic link to their public key file using the following commands in the bash console:
```

Comment:
no, it's not the `akka-http-signature` lib, it is `rww-play` that uses symbolic links to indicate the default representation for a document. I gave you a pointer to how this is done on other servers, and which explains the content-negotiation that occurs in http. You can do a quick explanation of that here. The point of such a paragraph would be to explain how this setup is complex and different for each type of server. We will in the next section then show how LDP gives us a way to do the same thing without knowing how the server is set up.